addons/account/models/account_move.py

```diff
@@ -1486,13 +1487,15 @@ def _compute_payments_widget_to_reconcile_info(self):
                     continue
 
                 payments_widget_vals['content'].append({
-                    'journal_name': line.ref or line.move_id.name,
+                    'journal_name': line.move_id.name,
```

Comment:
That does not make much sense to still call it journal_name then? (I would think it is the name of the journal)